README.md

```diff
@@ -2,47 +2,64 @@
 
-## Workflow options
+To use the GitHub Action with default settings:
 
-Change these options in the workflow `.yml` file to meet your GitHub project needs:
+1.  Set up an API token to allow the GitHub Action to authenticate on Codacy:
 
-| Setting                           | Description                                                                            | Recommended value                     |
-| --------------------------------- | -------------------------------------------------------------------------------------- | ------------------------------------- |
-| `api-token`                       | [Account API token](https://docs.codacy.com/codacy-api/api-tokens/#account-api-tokens) | `${{ secrets.CODACY_API_TOKEN }}`     |
-| `project-token`                   | [Project API token](https://docs.codacy.com/codacy-api/api-tokens/#project-api-tokens) | `${{ secrets.CODACY_PROJECT_TOKEN }}` |
-| `coverage-reports`                | Optional comma-separated list of reports to send                                       | `''`                                  |
-| `language`                        | Optionally force associating a language with your coverage report                      | `''`                                  |
-| `force-coverage-parser`           | Optionally force using a specific coverage report parser                               | `''`                                  |
+    -   **If you're setting up one repository**, [obtain a project API token](https://docs.codacy.com/codacy-api/api-tokens/#project-api-tokens) and store it as an [encrypted secret for your **repository**](https://docs.github.com/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `CODACY_PROJECT_TOKEN`.
+    -   **If you're setting up multiple repositories**, [obtain an account API token](https://docs.codacy.com/codacy-api/api-tokens/#account-api-tokens) and store it as an [encrypted secret for your **organization**](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization) with the name `CODACY_API_TOKEN`.
+
+    > ⚠️ **Never write API tokens to your configuration files** and keep your API tokens well protected, as they grant owner permissions to your projects on Codacy.
+
+2.  [Generate a supported code coverage report](https://docs.codacy.com/coverage-reporter/#adding-coverage-to-your-repository) before each push to your repository.
+
+3.  Add the following to a file `.github/workflows/codacy-coverage-reporter.yaml` in your repository. Update the sample code with the name of the environment storing the access token and the paths to the report files generated previously:
+
+    ```yaml
+    name: Codacy Coverage Reporter
+
+    on: ["push"]
+
+    jobs:
+      codacy-coverage-reporter:
+        runs-on: ubuntu-latest
+        name: codacy-coverage-reporter
+        environment: production # Environment storing the access token
```

Comment:
It's not mandatory to associate GitHub actions workflows with environments, as [this is an optional feature](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment). :thinking: So, my suggestion is to simplify the example and not mention this here:

```suggestion
```

I believe that you ran into this because in your tests you configured the secret `CODACY_PROJECT_TOKEN` under a ["sandbox" environment](https://github.com/codacy-docs/sandbox/settings/environments/729300222/edit). But you could have created a repository secret [directly here](https://github.com/codacy-docs/sandbox/settings/secrets/actions) without having to be associated with an environment. :slightly_smiling_face: